src/lib/Components/Contact.svelte

```diff
@@ -0,0 +1,83 @@
+<script>
+	import {
+		IconMailFilled,
+		IconPhoneFilled,
+		IconMapPinFilled,
+	} from '@tabler/icons-svelte';
+
+	let firstname = '';
+	let lastname = '';
+	let email = '';
+	let phone = '';
+	let message = '';
+
+	const handleSubmit = () => {
+		console.log('First Name:', firstname);
+		console.log('Last Name:', lastname);
+		console.log('Email:', email);
+		console.log('Phone:', phone);
+		console.log('Message:', message);
+	};
+</script>
+
+<section class="py-10 bg-surface-300">
+	<div class="grid justify-center gap-4 p-10 text-black">
+		<h1 class="font-bold text-center h1">Contact Us</h1>
+		<h3>Any questions or remarks? Just write us a message!</h3>
+	</div>
+	<div class="w-3/5 gap-4 p-1 mx-auto rounded-lg opacity-95 bg-slate-100 md:flex md:grid-cols-2">
```

Comment:
Can you increase contact section width on tablet view (md) and also increase width on mobile view please.